features/camera_effect.feature

```diff
@@ -0,0 +1,25 @@
+        Scenario: Camera effect implementation
+
+Feature: Camera effect validation
+                Given I get target
+                When I should see the numerical target
+                And I should see target number should be in the range of citizens list
+                And I should see target is relevant to alive citizens
+                Then validation method should return True
+
+Feature: Camera effect activation
+                Given activate method is called
+                Then active player should get message "Выберите гражданина, на дом которого будет установлена Видеокамера..."
+                And active player should enter the target number
+                When validation is successful
+                Then target number should be set as effect target
+                When validation is failed
+                Then activation should be repeated
+
+Feature: Camera effect resolve
+                Given resolve method is called and effect status is ACTIVATED
+                When kill, steal or stage effect is on the target citizen
+                Then player that activated kill, steal or stage effect should get message "Видеокамера зафиксировала как [роль игрока, который осуществил Убийство или Кражу] вломился в дом [целевой гражданин]Видеокамера зафиксировала как [роль игрока, который осуществил Убийство или Кражу] вломился в дом [целевой гражданин]"
```

Comment:
Maybe 'Player that activated Camera' - not the player that activated kill, ... ?

Reply:
Yes, my bad, i'll fix it, thanks)